TestFoundation/TestNSString.swift

```diff
@@ -81,7 +81,7 @@ class TestNSString : XCTestCase {
             ("test_ExternalRepresentation", test_ExternalRepresentation),
             ("test_mutableStringConstructor", test_mutableStringConstructor),
             ("test_PrefixSuffix", test_PrefixSuffix),
-            ("test_reflection", test_reflection),
+            ("test_reflection", { _ in test_reflection }),
```

Comment:
unrelated to this change but we should consider moving this into an actual test scope.